packages/markdown/remark/src/types.ts

```diff
@@ -1,13 +1,13 @@
 import type * as unified from 'unified';
-import type * as shiki from 'shiki';
+import type { ShikiConfig } from './remark-shiki';
 
 export type UnifiedPluginImport = Promise<{ default: unified.Plugin }>;
 export type Plugin = string | [string, any] | UnifiedPluginImport | [UnifiedPluginImport, any];
 
 export interface AstroMarkdownOptions {
 	mode?: 'md' | 'mdx';
 	syntaxHighlight?: 'prism' | 'shiki' | false;
-	shikiTheme?: shiki.Theme;
```

Comment:
does this break existing users? It would be good to leave shikiTheme support, so that existing users don't suddenly break.

Reply:
It won't be breaking because the previous PR hasn't been released yet (only to the preview channel)

Reply:
perfect, thanks for confirming